src/fdc3/dotnet/DesktopAgent.Shared/src/MorganStanley.ComposeUI.Fdc3.DesktopAgent.Shared/Contracts/CreatePrivateChannelResponse.cs

```diff
@@ -16,17 +16,10 @@ namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Shared.Contracts;
 
 internal class CreatePrivateChannelResponse
 {
-    private CreatePrivateChannelResponse(bool success, string? channelId, string? error)
-    {
-        Success = success;
-        ChannelId = channelId;
-        Error = error;
-    }
+    public bool Success { get; set; }
```

Comment:
I understand this change is due to serialization. Can we try if it works with "init set"?